src/Orchard.Web/Modules/Orchard.Widgets/Services/ILayerEvaluationService.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections.Generic;
+
+namespace Orchard.Widgets.Services
```

Comment:
Good idea would be merge interface definition with default implementation in single file as most services in Orchard.